.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "deno.enable": true
+}
```

Comment:
This file is no longer needed with a top-level `deno.json`. The extension enables itself automatically nowadays.